tests/testthat/test-clean_redcap.R

```diff
@@ -45,8 +45,8 @@ test_that("extract_nonrepeat_table tibble contains expected columns and data typ
   expect_character(out$text)
   expect_character(out$note)
   expect_double(out$calculated)
-  expect_double(out$dropdown_single)
-  expect_double(out$radio_single)
+  expect_character(out$dropdown_single)
+  expect_character(out$radio_single)
   expect_double(out$checkbox_multiple___1)
```

Comment:
Assuming this is for `label` - shouldn't `checkbox` and `truefalse` be `logical`? And `yesno` should be `factor`?

Reply:
Yes, I misread the issue and thought factor was only for the latter two (dropdown/radio). I can add an a quick coercion to the function that coerces to `factor`, I actually had it in an earlier commit and removed it.

Reply:
See above, these tests are based on `clean_redcap` and not the output of the new function. However, I did update the output of `yesno` in the newest commit.